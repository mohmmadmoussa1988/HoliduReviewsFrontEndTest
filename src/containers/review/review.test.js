import React from "react";
import { configure, shallow } from "enzyme";
import ReviewContainer from "./review";
import toJson from "enzyme-to-json";
describe(`<ReviewContainer />`, () => {
  it("should render ReviewContainer", () => {
    const wrapper = shallow(
      <ReviewContainer
        review={[
          {
            headline: "Amazing Pool!",
            comment:
              "Mauris justo ante, pulvinar eget consequat at, bibendum et lorem. Suspendisse lacus urna, fringilla sit amet commodo eget, condimentum et nisl. Pellentesque elit mi, porta in mi at, vulputate mattis lacus.",
            author: "Alissa Stacey",
            positiveFeedback: "The location is perfect",
            negativeFeedback: null,
            score: 4.1,
            channel: "AIRBNB",
            publishedAt: "2020-08-11T12:20:02.340Z",
          },
        ]}
      />
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
