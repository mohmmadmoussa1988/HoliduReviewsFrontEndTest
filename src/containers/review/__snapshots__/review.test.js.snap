// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReviewContainer /> should render ReviewContainer 1`] = `
<styled.div>
  <styled.div>
    <Score />
    <Channel />
  </styled.div>
  <styled.div>
    <styled.div />
  </styled.div>
  <styled.div>
    <styled.div />
  </styled.div>
  <styled.div>
    <Feeback />
  </styled.div>
  <styled.div>
    <styled.div />
  </styled.div>
  <styled.div>
    <styled.div>
      Reviewed 
      <n
        ago={false}
        calendar={false}
        decimal={false}
        element={null}
        filter={[Function]}
        format="DD MMMM YYYY"
        fromNow={false}
        interval={60000}
        local={false}
        onChange={[Function]}
        titleFormat=""
        toNow={false}
        trim={false}
        unit={null}
        unix={false}
        utc={false}
        withTitle={false}
      />
    </styled.div>
  </styled.div>
  <styled.hr />
</styled.div>
`;
